src/main/java/uk/nhs/hee/tis/trainee/forms/service/FormRPartAService.java

```diff
@@ -102,7 +112,12 @@ public FormRPartADto save(FormRPartADto formRPartADto) {
 
     // Forms stored in cloud are still stored to Mongo for backwards compatibility.
     formRPartA = repository.save(formRPartA);
-    return mapper.toDto(formRPartA);
+    FormRPartADto formDto = mapper.toDto(formRPartA);
+    if (formRPartA.getLifecycleState() == LifecycleState.SUBMITTED) {
+      log.debug("Publishing FormRPartA submitted event for form id: {}", formRPartA.getId());
+      eventBroadcastService.publishFormRPartAEvent(formDto, "formr-a", formRPartASubmittedTopic);
```

Comment:
The ndw-exporter needs to know the form type since this affects the datalake path. The only problem with this approach is that it uses the 'trigger' attribute, which was set up to indicate what sort of change for an LTFT form was triggering the event. Its not really appropriate for keeping the form type.